lib/websocket/driver/hybi/message.rb

```diff
@@ -14,7 +14,7 @@ def initialize
           @rsv2   = false
           @rsv3   = false
           @opcode = nil
-          @data   = Driver.encode('', :binary)
```

Comment:
can i know, what does it mean Opcode, and if i receive on client opcode -1 what does it mean, thx 

![image](https://cloud.githubusercontent.com/assets/132497/14105555/e1115cfe-f5de-11e5-9416-92091621eebb.png)